### ISSUE TITLE
Alerts not working

### ISSUE THREAD

**Issue by diegomartinmf** [EU-ECDC/epitweetr#71]
Hello
I am running epitweetr, and I could not solve some trouble about alerts
![image](https://user-images.githubusercontent.com/106124713/174667591-c3f5f884-e181-4b72-b418-5ac789d4f885.png)
Would like to ask if you could help me trying to identify errors.
I am writing from Uruguay. Thanks in advance. Kind regards

**Comment by forchard** [comment]
Hello dear charrua!
Let's see if we can help you with that!
Have you been able to launch the database  and data collection? Alerts cannot work without this features on.
Bets,
Francisco

**Comment by diegomartinmf** [comment]
Hello Francisco!
Thanks for your answer. And sorry I posted on issued. I may have posted it on Discussions perhaps.
Yes, started Database correctly, Data collection correctly. Everyghing seems to be working fine except the alerts task that start but gives an error as picture.
Sometimes database process stops but start again without my intervetion. The same with Data collection.
Now I added another language (English, because at beginning I only started with Spanish) and the process is taking too much time (also have finished with errors, when downloading file of language) (example: https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.en.300.vec.gz)
The error on alerts is "Cannot determine the last aggregated period for alert detection. "

Thanks in advance

**Comment by forchard** [comment]
Hi Diego
it seems that your installation is not aggregating data yet.
1. Can you see tweets on the Data Protection search feature?
2. Are you being able to see any data on the dashboard
3. Are you bein able to see geotraining examples on the Geo Tagging training page?

Thanks in advance
Francisco

**Comment by diegomartinmf** [comment]
Hello Francisco,

how are you doing?
1. Now, after several tries, I have information on Data Protection
2. The same, yes. After several tries, I have information on the dashboard
3. When I run the process, I see examples on that page too

I think I am having random errors that I cannot identify. Becasue some times the application seems to be running correctly, but in one moment it crashes 
I added some previous options, because I was having troubles when downloading files

The lines i run before starting are:
options(timeout=100000)
options(download.file.method = "auto")

This is one example of random error (it says Warning, but shiny section crashed after that)
![image](https://user-images.githubusercontent.com/106124713/177997881-11e54ce6-6069-471c-baa4-d49d41978f6e.png)

Thanks again for your help
Best regards
Diego

**Comment by forchard** [comment]
Hi Diego, 
Happy to hear you made it work! We know that installing and setting up epitweetr can be challenging. We are thinking into solutions to simplify it.

One question concerning the tasks.json tasks.  Is your epitweetr home on the same letter e.g. "c:\" than your windows temporary folder ?

Best,
Francisco

**Comment by forchard** [comment]
Great to hear! 
I manage to replicate the issue you have on the shiny app with the task.json and I am adding a fix for the next CRAN release.
I will close this issue.
You are welcome to create a new issue if you found new problems.
Best,
Francisco
